### DISCUSSION TITLE
How to reset Low traffic volume alarm setting displayed in infuxdb

### DISCUSSION THREAD

**Discussion by The1Waterman** [Security-Onion-Solutions/securityonion#12243] [General]
How can you reset the Low traffic volume alarm setting displayed in infuxdb, ie where is the threshold level is set.
> from(bucket: "_monitoring")
        :
> filter(fn: (r) => r["_level"] == "crit")

**Comment by reyesj2** [comment]
That alarm should be on when the traffic on your monitor interface is below 5mbps and it is checked at 1m intervals

**Comment by The1Waterman** [comment]
is there any way to chance this value?
